src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -3538,7 +3539,7 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
                 raise ex
     else:
         resource_groups.create_or_update(default_workspace_resource_group, {
```

Comment:
Not all `create_or_update` are changed to `begin_create_or_update` in track 2, only LRO (long running operation) is changed to `begin_xxx_xxx_xxx`, you need check SDK source code as source of truth.

Reply:
sure, but it seems that there is no ```create_or_update ``` method in "operations/_managed_clusters_operations.py" in azure-mgmt-containerservice-15.1.0 any more.

Reply:
It becomes `begin_creat_or_update` now.